### ISSUE TITLE
Issues on sfz v1 opcodes

### ISSUE THREAD

**Issue by kinwie** [sfztools/sfizz#433]
These opcodes has issues in sfizz :

- [x] `loop_sustain`
Current implementation is only put the region to loop_continuous. But loop_sustain is different.

- [x] `ampeg_xxx` with `delay` defined _(also with `delay_random`)_
The envelope phase is not delayed by the specified delay time. It just start from note-on only
Ex :
```
<region>
sample=*sine
ampeg_attack=0.5
ampeg_hold=0.5
ampeg_decay=0.5
ampeg_sustain=0
ampeg_release=1
delay=1
```

- [x] `pitch_veltrack`
Pitch_veltrack curve not yet working properly.
changes only at vel 127. Vel 1-126 not affected

**Comment by paulfd** [comment]
Ho I'm actually not sure we handle delay *at all*.

**Comment by kinwie** [comment]
Yes, pitch_veltrack is working fine now :+1:

**Comment by kinwie** [comment]
Tested the new build. `loop_sustain` is working fine now :+1:

**Comment by paulfd** [comment]
All of this is solved now hopefully, the delay one was in #655, the loop sustain in #654.
